<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="1" total="2" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-19T12:34:39Z" name="N11 Test Suite" finished-at="2023-11-19T12:34:51Z" duration-ms="11349">
    <groups>
    </groups>
    <test started-at="2023-11-19T12:34:39Z" name="LoginTest" finished-at="2023-11-19T12:34:51Z" duration-ms="11349">
      <class name="com.nopecommerce.scripts.LoginFeature">
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.nopecommerce.scripts.LoginFeature@6025e1b6]" started-at="2023-11-19T18:04:39Z" name="setUp" finished-at="2023-11-19T18:04:47Z" duration-ms="7331" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="logintestcase_1001()[pri:0, instance:com.nopecommerce.scripts.LoginFeature@6025e1b6]" started-at="2023-11-19T18:04:47Z" name="logintestcase_1001" finished-at="2023-11-19T18:04:50Z" duration-ms="3697" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Login was unsuccessful. Please correct the errors and try again. 
No customer account found] but found [Login was unsuccessful. Please correct the errors and try again.
No customer account found]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Login was unsuccessful. Please correct the errors and try again. 
No customer account found] but found [Login was unsuccessful. Please correct the errors and try again.
No customer account found]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.nopecommerce.scripts.LoginFeature.logintestcase_1001(LoginFeature.java:30)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logintestcase_1001 -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.nopecommerce.scripts.LoginFeature@6025e1b6]" started-at="2023-11-19T18:04:50Z" name="tearDown" finished-at="2023-11-19T18:04:51Z" duration-ms="291" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.nopecommerce.scripts.LoginFeature -->
    </test> <!-- LoginTest -->
  </suite> <!-- N11 Test Suite -->
</testng-results>
